app/helpers/common/priceFormatter.js

```diff
@@ -1,16 +1,8 @@
-export const formatPrice = (priceValue) => {
-  const formattedPrice = priceValue.toLocaleString(undefined, {
-    minimumFractionDigits: 3, maximumFractionDigits: 3,
-  });
+export const formatNumber = ({ value, minimumFractionDigits = 0, maximumFractionDigits = 4 }) => (
+  parseFloat(value)
+    .toLocaleString(
+      undefined, { minimumFractionDigits, maximumFractionDigits },
+    )
+);
 
```

Comment:
I don't love this format but I really had to try hard to find something eslint agreed with.

It is currently setup to not like line-breaks for arrow functions but insists on 100 character max line length so wrapping in brackets is necessary.